### PR TITLE
Update gulpfile to use exports instead of gulp.task syntax

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Proposed Changes

Update gulpfiles.js to use export instead of `gulp.task()` syntax.
### Reason for Changes
Prep for breaking the gulpfile apart by type of task.

### Test Coverage
Ran npm run build.

### Additional Information

There were a lot of tasks defined with `gulp.task` that I chose to not explicitly export, since we aren't exposing them in our `package.json`.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -854,55 +854,51 @@ gulp.task('package-json', function(cb) {
...
-  'package-locales',
-  'package-media',
-  'package-umd-bundle',
-  'package-json',
-  'package-readme',
-  'package-dts'
-  ));
-
-// The default task builds Blockly.
-gulp.task('default', gulp.series(['build']));
```
Comment (samelhusseini) [inline gulpfile.js]: What happens to default? ie: when someone runs ``gulp``.

Comment (rachel-fenichel) [reply gulpfile.js]: I'm exporting default: build at the bottom.

File: gulpfile.js [hunk 2]
```diff
@@ -941,65 +941,77 @@ function getRCBranchName() {
...
-    'typings',
-    function(done) {
-      console.log('push-rebuild-branch: committing rebuild');
-      execSync('git commit -am "Rebuild"', { stdio: 'inherit' });
-      var branchName = getRebuildBranchName();
-      execSync('git push origin ' + branchName, { stdio: 'inherit' });
-      console.log('Branch ' + branchName + ' pushed to GitHub.');
-      console.log('Next step: create a pull request against develop.');
-      done();
+const recompile = gulp.series(
```
Comment (samelhusseini) [inline gulpfile.js]: I would prefer if we prefaced all the git related things with git, ``gitRecompile`` for this as a form of grouping.

Comment (rachel-fenichel) [reply gulpfile.js]: Discussed offline--they're still grouped in the exports.

**Pull request merged**